openedx/features/course_experience/templates/course_experience/course-outline-fragment.html

```diff
@@ -107,7 +101,6 @@ <h3 class="section-title">${ section['display_name'] }</h3>
                                     <div class="subsection-actions">
                                         ## Resume button (if last visited section)
                                         % if subsection['last_accessed']:
-                                            <span class="sr-only">${ _("This is your last visited course section.") }</span>
```

Comment:
Is this just functionality we are cutting? or is there another way for screen reader users to know which section is their last visited?

Reply:
Mark suggested dropping it in the bug:

"""
Note, I suggest removing the extra sr-only text that is added before the Resume Course text, its unnecessary and adds extra verbiage to an already very long link text. In addition to simplifying the HTML as per below, remove the following node:

<span class="sr-only">This is your last visited course section.</span>
"""

Reply:
Thanks for double-checking this @Mjloturco.  The text is very verbose, so I made the decision to remove it.  The text "Resume course" is accessible to all users, including SR users, and is sufficient for informing all users that the link takes them to where they left off.  As a general rule, I am trying to reduce extra verbosity where it is not required.